main.go

```diff
@@ -5,11 +5,47 @@
 package main
 
 import (
-	"github.com/harness/harness-docker-runner/cli"
+	"runtime"
 
+	"github.com/harness/harness-docker-runner/cli"
 	_ "github.com/joho/godotenv/autoload"
+	"github.com/kardianos/service"
 )
 
 func main() {
-	cli.Command()
+	if runtime.GOOS == "windows" {
```

Comment:
just make sure that this works for both msi way and non-msi way

Reply:
yes, it works directly with .exe as well. Tested this with Devki